.buildkite/hooks/pre-command

```diff
@@ -43,3 +43,18 @@ export GRADLE_BUILD_CACHE_USERNAME
 
 GRADLE_BUILD_CACHE_PASSWORD=$(vault read -field=password secret/ci/elastic-elasticsearch/migrated/gradle-build-cache)
 export GRADLE_BUILD_CACHE_PASSWORD
+
+BUILDKITE_API_TOKEN=$(vault read -field=token secret/ci/elastic-elasticsearch/buildkite-api-token)
```

Comment:
Could we just use `BUILDKITE_AGENT_ACCESS_TOKEN` instead?

Reply:
Nope, those are special tokens issued by Buildkite that are only used for the `buildkite-agent` CLI commands